libs/sdk-ui-theme-provider/src/cssProperties.ts

```diff
@@ -5,7 +5,6 @@ import { IThemePalette, ITheme } from "@gooddata/sdk-backend-spi";
 
 // keep it in sync with SCSS:$gd-color-text-light
 const GD_COLOR_TEXT_LIGHT = "#fff";
```

Comment:
both commits should have commit msg according contribution doc:
"Also note how to add JIRA ticket reference: using a "JIRA: [ticket-id]" to the end of the commit body"